.github/workflows/hz-enterprise-op.-rhel-release.yaml

```
@@ -185,10 +189,9 @@ jobs:
           sed -i  "s|image: .*|image: ${RHEL_IMAGE}|" ./${OPERATOR_NAME}/operator-rhel.yaml
           sed -i  "s/productVersion: .*/productVersion: ${OPERATOR_VERSION}/g" ./${OPERATOR_NAME}/operator-rhel.yaml
 
-          linesArray=($(sed -n  "/value: .*/=" ./${OPERATOR_NAME}/operator-rhel.yaml))
-          sed -i "${linesArray[2]}s/value: .*/value: \"${HAZELCAST_VERSION}\"/" ./${OPERATOR_NAME}/operator-rhel.yaml
-          sed -i "${linesArray[4]}s/value: .*/value: \"${MANCENTER_VERSION}\"/" ./${OPERATOR_NAME}/operator-rhel.yaml
-        env: 
+          sed -i "s|${REGISTRY_NAME}/hazelcast-enterprise-4-rhel8:.*|${REGISTRY_NAME}/hazelcast-enterprise-4-rhel8:${HAZELCAST_VERSION}|" ./${OPERATOR_NAME}/operator-rhel.yaml
+          sed -i "s|${REGISTRY_NAME}/management-center-4-rhel8:.*|${REGISTRY_NAME}/management-center-4-rhel8:${MANCENTER_VERSION}|" ./${OPERATOR_NAME}/operator-rhel.yaml
```

```suggestion
          sed -i "s|${RHEL_HAZELCAST_REPO}:.*|${RHEL_HAZELCAST_REPO}:${HAZELCAST_VERSION}|" ./${OPERATOR_NAME}/operator-rhel.yaml
          sed -i "s|${RHEL_MANCENTER_REPO}:.*|${RHEL_MANCENTER_REPO}:${MANCENTER_VERSION}|" ./${OPERATOR_NAME}/operator-rhel.yaml
```
`REGISTRY_NAME` is not defined in this step of before.